PyZOGY/__main__.py

```diff
@@ -41,7 +41,7 @@ def main():
     parser.add_argument('--log', dest='log', help='Log output file', default='pyzogy.log')
     parser.add_argument('--percent', dest='percent', help='Pixel percentile for gain matching on pixels',
                         default=99)
-
+    parser.add_argument('--pixstack-limit', dest='pixstack_limit', type=int, help='Modify set_extract_pixstack in Sep', default=None)
```

Comment:
Can you remove the `dest='pixstack_limit'` and 'default=None` arguments? These are already the defaults in `argparse`. Otherwise looks great!

Reply:
Done and retested